contrib/docker/Dockerfile.jdbc.build

```diff
@@ -1,18 +1,10 @@
-FROM openjdk:8
+FROM maven:3.6.3-adoptopenjdk-8
 
-# Download maven
-RUN \
-  wget https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.tar.gz && \
-  tar -xvf apache-maven-3.5.0-bin.tar.gz -C /bin && \
-  mv /bin/apache-maven-3.5.0 bin/maven
+RUN apt-get update \
+    && apt-get install -y protobuf-compiler  \
+    && apt-get clean
 
 # Sets maven to use a useful settings.xml file
 COPY \
   contrib/docker/maven-settings.xml /bin/maven/conf/settings.xml
 
-# Install protobuf 3.2 and give all users exec access
```

Comment:
I think the version of the protobuf compiler needs to match [the version in the pom file](https://github.com/bloomberg/comdb2/blob/master/cdb2jdbc/pom.xml#L30). Protobuf isn't particularly good at backward compatibility. See the build failure [here](http://comdb2.s3-website-us-east-1.amazonaws.com/tests/0145711c/smoketest.log).

Reply:
Noted.

I bumped the library version to 3.8.0.

Reply:
I actually prefer to lock the version of protobuf. The version of protobuf-compiler from apt-get may very well change in the future. It'll then likely break the build again. With that said, I'm fine with using maven from apt-get (it didn't exist when the Dockerfile was first written), and leaving the protobuf installation as is.